server/core/src/https/views/login.rs

```diff
@@ -90,6 +90,30 @@ pub async fn view_index_get(
     }
 }
 
+// Maybe cursed ?
+pub async fn view_logout_get(
+    State(state): State<ServerState>,
+    VerifiedClientInformation(_client_auth_info): VerifiedClientInformation,
+    Extension(_kopid): Extension<KOpId>,
+    _jar: CookieJar,
+) -> Response {
+    let mut bearer_cookie = Cookie::new(COOKIE_BEARER_TOKEN, "");
```

Comment:
It seems weird that just deleting the cookie isn't working, because it certainly works in other parts of the auth flow. Are you sure this is needed?

Reply:
Yeah, path seems to be required in my testing on firefox, `bearer=; Path=/; Expires=Thu, 04 Jul 2024 15:15:46 GMT` deletes the cookie for me.

The cookiejar remove method: `bearer=; Max-Age=0; Expires=Wed, 05 Jul 2023 15:57:02 GMT`  does not

> because it certainly works in other parts of the auth flow

Probably because bearer_cookie.set_path("/"); is set causes this different behavior compared to `auth-session-id`

Reply:
That seems like a weird issue and one we should look into because we need to be able to use .remove.